src/pages/Profile/subpages/Settings/Basics.js

```diff
@@ -9,19 +9,19 @@ const genderOptions = [
   "male",
   "female",
   "transgender female",
-  "transgender male"
+  "transgender male",
```

Comment:
Less about this PR/code, but we may want to revisit this list based on existing standards - e.g., Facebook's: https://abcnews.go.com/blogs/headlines/2014/02/heres-a-list-of-58-gender-options-for-facebook-users